<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorWindow</class>
 <widget class="QDialog" name="AdministratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="UserTableView">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>751</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="CreditTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>541</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="DepositTableView">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>601</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="BankTableView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>620</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
